importer/src/main/java/org/hiero/mirror/importer/downloader/block/StreamProperties.java

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hiero.mirror.importer.downloader.block;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import java.time.Duration;
+import lombok.Data;
+import org.hibernate.validator.constraints.time.DurationMin;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+public class StreamProperties {
+
+    @Min(1000)
+    private int maxBlockItems = 800_000;
+
+    @Min(1)
+    private int maxStreamResponseSizeMB = 8;
```

Comment:
Prefer the use of DataSize
```suggestion
    @NotNull
    @DataSizeUnit(DataUnit.KILOBYTES)
    private DataSize maxStreamResponseSize = DataSize.ofMegabytes(8);
```